src/lib/queryFromFilters.js

```diff
@@ -21,6 +21,13 @@ export default function queryFromFilters(className, filters) {
   return query;
 }
 
+function addQueryConstraintFromObject(query, filter, constraintType) {
+  let compareTo = JSON.parse(filter.get('compareTo'));
+  for (let key of Object.keys(compareTo)) {
+    query[constraintType](filter.get('field')+'.'+key, compareTo[key]);
```

Comment:
It's an old javascript feature, I'm just accessing the function by its name through bracket notation https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_Accessors#Bracket_notation